CMakeLists.txt

```diff
@@ -35,3 +35,8 @@ install(FILES cmakesample.desktop
 install(FILES cmakesample.png
     DESTINATION share/icons/hicolor/86x86/apps
 )
+
+# Tell Qt Creator where the application executable would be located on the device.
+# See https://doc.qt.io/qtcreator/creator-deployment-embedded-linux.html#deploying-cmake-projects-to-embedded-linux-devices
```

Comment:
Like mentioned in IRC, I think it would be worth mentioning that the file is not used for actually deploying anything

Reply:
Updated